mypyc/ops_list.py

```diff
@@ -54,6 +54,14 @@ def emit_new(emitter: EmitterInterface, args: List[str], dest: str) -> None:
     emit=simple_emit('{dest} = CPyList_GetItemShort({args[0]}, {args[1]});'),
     priority=2)
 
+list_get_item_unsafe_op = custom_op(
```

Comment:
Add comment describing in which ways this is unsafe. I believe all of these apply:
1. no index overflow check 
2. assume non-negative index 
3. assume short integer index